src/aura/RD2_EnablementDelegateSimple/RD2_EnablementDelegateSimple.cmp

```diff
@@ -0,0 +1,222 @@
+                                        <div class="slds-is-relative slds-m-right_large">
+                                            <lightning:spinner aura:id="metadeployConfirmSpinner" size="small"
+                                                variant="brand" alternativeText="{!$Label.c.labelMessageLoading}"
+                                                class="slds-hide" />
+                                        </div>
+                                        <lightning:input type="checkbox"
+                                            label="{!$Label.c.RD2_EnablementMetaDeployConfirm}" variant="label-hidden"
+                                            title="{!$Label.c.RD2_EnablementMetaDeployConfirm}"
+                                            aura:id="metadeploy-confirm" name="metadeploy-confirm"
+                                            checked="{!v.state.isMetaConfirmed}" onchange="{!c.handleMetaDeployConfirm}"
+                                            disabled="{!or(not(v.state.isMetaLaunched), v.state.isMetaConfirmed)}" />
+                                    </div>
+                                </div>
+                            </li>
+
+                        </ol>
+                        <div aria-valuemin="0" aria-valuemax="100" aria-valuenow="{!v.state.metaDeployProgress}"
+                            role="progressbar">
+                            <span class="slds-assistive-text">{!$Label.c.ProgressMarkerAssistiveTextProgress + ' ' +
+                                v.state.metaDeployProgress + '%'}</span>
+                        </div>
+                    </div>
+
+                    <aura:if isTrue="{!equals(v.errorSection, 'metadeploy')}">
+                        <lightning:layout>
+                            <lightning:layoutItem size="9" class="slds-p-left_medium">
+                                <c:pageNotification severity="error" size="small" message="{!v.errorMessage}" />
+                            </lightning:layoutItem>
+                        </lightning:layout>
+                    </aura:if>
+
+                    <div class="slds-p-bottom_x-small" />
+
+                </div>
+            </lightning:card>
+        </lightning:layoutItem>
+        <lightning:layoutItem padding="horizontal-small" size="12">
+            <lightning:layout horizontalAlign="spread" multipleRows="true">
+                <lightning:card class="pageSection" aura:id="succuessNextStep">
+                    <lightning:layoutItem class="slds-p-left_medium" size="9">
+                        <aura:if isTrue="{!v.state.isMetaConfirmed}">
+                            <lightning:formattedRichText
+                                value="{!$Label.c.RD2_EnablementMigrationCompleteMessage}" />
+                        </aura:if>
+                    </lightning:layoutItem>
+                    <lightning:layoutItem size="3" />
+
+                    <lightning:layoutItem size="9" >
```

Comment:
- We do not need error section in the success message layout.
- The border for the section does not go to the end of the page like the previous two sections (the screenshot has been attached to the WI).